models/eos/route_maps/eos_route_maps.yml

```diff
@@ -0,0 +1,391 @@
+module: eos_route_maps
+short_description: Manages Route Maps resource module
+description: This module configures and manages the attributes of Route Mapd on Arista
+  EOS platforms.
+  version_added: 2.0.0
+  author: Gomathi Selvi Srinivasan (@GomathiselviS)
+  notes:
+  - Tested against Arista EOS 4.23.0F
+  - This module works with connection C(network_cli). See the L(EOS Platform Options,eos_platform_options).
+  options:
+    config:
+      description: A list of route-map options
+      type: list
+      elements: dict
+      suboptions:
+        route_maps:
+          description:
+          - Route Map statements.
+          type: list
+          elements: dict
+          suboptions:
+            route_map_name:
```

Comment:
Can this be moved directly under config? For example:
```yaml
- nxos_route_maps:
    config:
      - route_map: rmap1
        entries:
          - sequence: 10
            action: permit
            description: test-1
          - sequence: 11
            action: deny
            description: test-2
      - route_map: rmap2
        entries:
          - sequence: 11
            action: permit
            description: test-1
          - sequence: 12
            action: deny
            description: test-2
```

Reply:
> Can this be moved directly under config? For example:
> 
> ```yaml
> - nxos_route_maps:
>     config:
>       - route_map: rmap1
>         entries:
>           - sequence: 10
>             action: permit
>             description: test-1
>           - sequence: 11
>             action: deny
>             description: test-2
>       - route_map: rmap2
>         entries:
>           - sequence: 11
>             action: permit
>             description: test-1
>           - sequence: 12
>             action: deny
>             description: test-2
> ```

Yes you are right.